cli/Valet/WSL.php

```diff
@@ -1,103 +0,0 @@
-<?php
```

Comment:
This could be reworked to export the CA to windows where it will need to be trusted.